common/SmokeTests/SmokeTest/CosmosDBTest.cs

```diff
@@ -130,7 +130,7 @@ private static async Task CreateDocuments()
         /// <summary>
         /// The query retrieve all planets, this is going to verify that planets match
         /// </summary>
-        private static async Task ExecuteSimpleQuery(){
+        private static void ExecuteSimpleQuery(){
```

Comment:
I hope you'll forgive my taking the liberty of clearing out some warnings for these.